src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,122 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private final int defaultSize = 10;
```

Comment:
uppercase for constants

Reply:
**Error:  /home/runner/work/jv-array-list/jv-array-list/src/main/java/core/basesyntax/ArrayList.java:6:23: Abbreviation in name 'DEFAULTCAPACITY' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]
Error:  /home/runner/work/jv-array-list/jv-array-list/src/main/java/core/basesyntax/ArrayList.java:6:23: Member name 'DEFAULTCAPACITY' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'. [MemberName]**